<?xml version="1.0" encoding="UTF-8"?>
<launch>

<<<<<<< HEAD
 <!--include file="$(find pocketsphinx)/demo/robocup_2.launch"> </include-->
 <!--include file="$(find darknet_ros)/launch/darknet_ros.launch"> </include-->


 
 <node pkg="god_states_machine" name="main_node" type="main_sm.py" output="screen" launch-prefix="xterm -e"/>
 <!--speech-->
        <node pkg="smach_viewer" name="structure_state_machine" type="smach_viewer.py"/>
	<node name="HotwordRecogniser" pkg="tum_alle_athome_speech_hotword" type="hotword-detect.py" output="screen">
		<param name="hotword_model_tiago" value="$(find tum_alle_athome_speech_hotword)/resources/Tiago.pmdl"/>
		<param name="hotword_model_say" value="$(find tum_alle_athome_speech_hotword)/resources/Say.pmdl"/>
		<param name="hotword_model_move" value="$(find tum_alle_athome_speech_hotword)/resources/Move.pmdl"/>
		<param name="hotword_model_follow" value="$(find tum_alle_athome_speech_hotword)/resources/Follow.pmdl"/>
		<param name="hotword_model_bring" value="$(find tum_alle_athome_speech_hotword)/resources/Bring.pmdl"/>
		<param name="hotword_model_start" value="$(find tum_alle_athome_speech_hotword)/resources/Start.pmdl"/>
		<param name="hotword_model_wait" value="$(find tum_alle_athome_speech_hotword)/resources/Wait.pmdl"/>
		<param name="hotword_model_stop" value="$(find tum_alle_athome_speech_hotword)/resources/Stop.pmdl"/>
		<param name="hotword_model_yes" value="$(find tum_alle_athome_speech_hotword)/resources/Yes.pmdl"/>
		<param name="hotword_model_no" value="$(find tum_alle_athome_speech_hotword)/resources/No.pmdl"/>
		<param name="commands_sensitivity" value="0.5"/>
		<param name="tiago_sensitivity" value="0.5"/>
		    <!--param name="speech_recognition_srv" value="/tum_alle_athome_speech_recognition/speech_recog"/-->
		<param name="speech_recognition_srv" value="/planner"/>
    </node>
   <node name="speech_node" pkg="tum_alle_athome_speech_tts" type="text_to_speech.py" output="screen" />
   
  <node name="simple_navigation_goals" pkg="navigation" type="simple_navigation_goals" output="screen" args="-d $(find navigation)/src/simple_navigation_goals.cpp" />
   
        <!--Manipulation-->
<!--
  <rosparam command="load" file="$(find manipulation)/config/pick_motions.yaml" /> 

  
  <node name="pick_and_place_server" pkg="manipulation" type="pick_and_place_server.py" output="screen">
      <rosparam command="load" file="$(find manipulation)/config/pick_and_place_params.yaml" />
      <param name="object_width"  value="0.01" />
      <param name="object_height" value="0.06" />
      <param name="object_depth"  value="0.06" />
  </node>

  <node name="pick_client" pkg="manipulation" type="pick_client.py" output="screen"/>  
  <node name="object_detection" type="object_detection" pkg="perception" output="screen"/> Pick & place server -->

=======
<arg name="rviz"     default="true"/>
 <arg name="public_sim"   default="false"/>
 <arg name="sim_sufix" value="_public_sim"     if="$(arg public_sim)"/>
  <arg name="sim_sufix" value=""                unless="$(arg public_sim)"/>

<node name="rviz" pkg="rviz" type="rviz" if="$(arg rviz)"
          args="-d $(find tiago_2dnav)/config/rviz/navigation_public_sim.rviz"/>
>>>>>>> cc9392bc67e44b10c1c80bc4ba0a97209018581e

</launch>
